gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/SelectableAvatar.kt

```diff
@@ -0,0 +1,67 @@
+package com.gravatar.quickeditor.ui.components
+
+import android.util.Log
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.gravatar.quickeditor.R
+import com.gravatar.ui.GravatarTheme
+
+@Composable
+internal fun SelectableAvatar(
+    imageUrl: String,
+    isSelected: Boolean,
+    onAvatarClicked: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val cornerRadius = 7.dp
+    GravatarTheme {
+        AsyncImage(
+            model = imageUrl,
+            contentDescription = stringResource(id = R.string.selectable_avatar_content_description),
+            modifier = modifier
+                .clip(RoundedCornerShape(cornerRadius))
+                .then(
+                    if (isSelected) {
+                        Modifier.border(7.dp, MaterialTheme.colorScheme.primary, RoundedCornerShape(cornerRadius))
+                    } else {
+                        Modifier.border(2.dp, MaterialTheme.colorScheme.surfaceDim, RoundedCornerShape(cornerRadius))
+                    },
+                )
+                .clickable {
+                    onAvatarClicked()
+                },
+        )
+    }
+}
+
+@Preview
+@Composable
+private fun SelectableAvatarNotSelectedPreview() {
+    SelectableAvatar(
+        "https://gravatar.com/avatar/fd2188b818f15e629f7b62896b5c6075?s=250",
+        false,
+        { Log.i("SelectableAvatar", "Clicked") },
```

Comment:
Do you want to keep the log in the Preview?

Reply:
I've used it when I run the `@Preview` on a device, but I can remove it. It doesn't add much value.